tests/kernel/qa_test_klp.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Test for kernel's KLP infrastructure
```

Comment:
Only a minor nit, but you should write KLP out in full (kernel live patching?)

Reply:
ook